plugins/bounce.js

```diff
@@ -370,22 +370,22 @@ exports.bounce_spf = function (next, connection) {
                     connection.logerror(plugin, err.message);
                     return run_cb();
                 }
-                connection.logdebug(plugin, 'ip=' + ip + ' ' +
-                                            'spf_result=' + spf.result(result));
+                connection.logdebug(plugin, `ip=${ip} \
+                spf_result=${spf.result(result)}`);
```

Comment:
It's a judgement call, and this is okay, but I think it would be better on one line. Thoughts?

Reply:
I agree however I was just trying to keep consistent with the original formatting.